.github/workflows/publish.yml

```
@@ -73,9 +77,16 @@ jobs:
 
       - name: publish
         shell: pwsh
+        env:
+          RELEASE_TAG: ${{ github.event.inputs.release_tag }}
         working-directory: src/D2L.Bmx
         run: |
-          dotnet publish -r $env:PLATFORM-x64 -o build
+          $version = "$env:RELEASE_TAG"
+          if ( -not $version -or $version -eq '' ){
```

empty string is falsy in PowerShell
```suggestion
          if ( -not $version ){
```